### ISSUE TITLE
✅ ♿ 🐞 Bug Report: Issue with ARIA on mega menus

### ISSUE THREAD

**Issue by fentisam** [elementor/elementor#22554]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

There is an issue with the new mega menu feature as it is not giving divs their own custom aira tag.

### Steps to reproduce

1) Create a mega menu in the header
2) Add a WordPress menu to sit inside of the mega menu.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.4.26
	PHP Version: 7.4.33
	PHP Memory Limit: 128M
	PHP Max Input Vars: 2500
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://pro-dek.co.uk
	Home URL: https://pro-dek.co.uk
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 512M
	Max Memory limit: 256M
	Permalink Structure: /%category%/%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: kieron.salt@fenti.co.uk
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
	ACF Photo Gallery Field
		Version: 1.8.1
		Author: Navneil Naicker

	Admin Columns
		Version: 4.6.1
		Author: AdminColumns.com

	Advanced Custom Fields
		Version: 6.1.4
		Author: WP Engine

	Auto Image Attributes From Filename With Bulk Updater
		Version: 4.3.1
		Author: Arun Basil Lal

	Auto Image Attributes Pro
		Version: 2.0
		Author: Arun Basil Lal

	Disable Comments
		Version: 2.4.3
		Author: WPDeveloper

	Duplicator Pro
		Version: 4.0.1
		Author: Snap Creek

	Elementor
		Version: 3.13.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.1
		Author: Elementor.com

	Essential Addons for Elementor
		Version: 5.7.2
		Author: WPDeveloper

	Essential Addons for Elementor - Pro
		Version: 4.4.11
		Author: WPDeveloper

	GTM4WP
		Version: 1.16.2
		Author: Thomas Geiger

	Image optimization service by Optimole
		Version: 3.6.1
		Author: Optimole

	Link Whisper
		Version: 2.2.3
		Author: Link Whisper

	OMGF
		Version: 5.5.6
		Author: Daan from Daan.dev

	Rank Math SEO
		Version: 1.0.115
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.28
		Author: Rank Math

	Safe SVG
		Version: 2.1.1
		Author: 10up

	Search Regex
		Version: 3.0.6
		Author: John Godley

	Velvet Blues Update URLs
		Version: 3.2.10
		Author: VelvetBlues.com

	White Label CMS
		Version: 2.5
		Author: www.videousermanuals.com

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence

	WPCode Lite
		Version: 2.0.10
		Author: WPCode

	WP Rocket
		Version: 3.13.1
		Author: WP Media

	WP Rocket | Enforce Trailing Slash on URLs
		Version: 
		Author: WP Rocket Support Team

	WPS Hide Login
		Version: 1.9.8
		Author: WPServeur, NicolasKulka, wpformation


== Must-Use Plugins ==
	StackCache
		Version: 
		Author: Stack CP


== Elements Usage ==
	
	container : 0
		breadcrumbs : 6
		button : 15
		container : 8
		divider : 15
		eael-divider : 7
		heading : 16
		html : 1
		image : 33
		spacer : 1
		tabs : 1
		text-editor : 62
		video : 4
	footer : 8
		button : 2
		container : 1
		divider : 1
		eael-divider : 1
		form : 7
		heading : 2
		image : 5
		posts : 2
		text-editor : 12
	header : 1
		button : 1
		container : 17
		heading : 7
		image : 2
		mega-menu : 1
		nav-menu : 10
		search-form : 2
	page : 37
		button : 73
		divider : 42
		heading : 17
		image : 34
		image-carousel : 6
		text-editor : 101
	section : 18
		button : 17
		heading : 8
		image : 7
		text-editor : 13
	wp-post : 23
		button : 22
		counter : 24
		eael-advanced-data-table : 1
		heading : 7
		icon-box : 5
		icon-list : 5
		image : 64
		testimonial : 5
		text-editor : 68
		video : 1
	wp-page : 43
		breadcrumbs : 3
		button : 67
		divider : 51
		eael-advanced-data-table : 1
		google_maps : 1
		heading : 67
		html : 3
		icon-list : 17
		image : 48
		image-carousel : 2
		menu-anchor : 4
		shortcode : 1
		tabs : 1
		text-editor : 125
		video : 9


== Settings ==
	
	cpt_support: post, page
	disable_typography_schemes: yes
	allow_tracking: yes
	css_print_method: internal
	unfiltered_files_upload: 1


== Features ==
	Custom Fonts: 2
	Custom Icons: 0

== Integrations ==
	
	recaptcha: Active
	recaptcha_v3: Active


== Elementor Experiments ==
	Optimised DOM Output: Active
	Improved asset loading: Active
	Improved CSS Loading: Inactive
	Inline Font Icons: Active
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Editor Top Bar: Active
	Landing pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active
	Scroll Snap: Active by default
	Menu: Active


== Log ==
	
Log: showing 20 of 312023-05-02 12:51:18 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-02 12:51:21 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-02 12:51:21 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-02 12:51:23 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-02 12:51:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-05-02 12:51:25 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-05-02 12:51:27 [info] elementor-pro::elementor_pro_updater Started 
2023-05-02 12:51:27 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-02 12:51:27 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-02 12:51:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.3',
)]
2023-05-02 12:51:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.3',
)]
2023-05-19 09:57:00 [info] elementor::elementor_updater Started 
2023-05-19 09:57:00 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-19 09:57:03 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-19 09:57:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.2',
)]
2023-05-19 09:57:05 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.2',
)]
2023-05-19 09:57:07 [info] elementor-pro::elementor_pro_updater Started 
2023-05-19 09:57:07 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-19 09:57:07 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-19 09:57:07 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.1',
)]

PHP: showing 7 of 7PHP: 2023-04-06 14:20:09 [notice X 29][/home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-06 14:20:10 [notice X 81][/home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: /home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-04-06 14:25:23 [notice X 2][/home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor/includes/template-library/sources/local.php::747] ob_end_clean(): failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-13 12:35:22 [notice X 3][/home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor/includes/managers/image.php::80] Undefined offset: 2 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-13 22:51:44 [notice X 48][/home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::265] Trying to get property 'ID' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-17 10:15:41 [notice X 17][/home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor/app/modules/import-export/module.php::695] Trying to get property 'label' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-25 11:45:36 [notice X 1][/home/sites/10b/d/dbc01efc07/public_html/wp-content/plugins/elementor-pro/modules/forms/classes/recaptcha-v3-handler.php::124] Undefined index: score [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 4 of 4JS: 2023-05-16 10:14:52 [error X 1][https://pro-dek.co.uk/wp-includes/js/dist/vendor/react-dom.min.js?ver=18.2.0:10:23751] Cannot read properties of null (reading 'getOptions') 
JS: 2023-05-17 13:15:04 [error X 1][https://pro-dek.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:1081263] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-05-17 13:24:29 [error X 1][https://pro-dek.co.uk/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view2733") has already been destroyed and cannot be used. 
JS: 2023-05-17 13:31:18 [error X 1][https://pro-dek.co.uk/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view2445") has already been destroyed and cannot be used. 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Essential Addons for Elementor: Compatible
	Essential Addons for Elementor - Pro: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Essential Addons for Elementor: Compatible
	Essential Addons for Elementor - Pro: Incompatible
				
````

</details>

**Comment by nicholaszein** [comment]
Hello @fentisam! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by fentisam** [comment]
@hein-obox It's with the class named "e-n-menu-items-content" and "e-n-menu-item-title e-normal".

I have run the site through PageSpeed Insights and it is showing the errors on desktop:
[https://pagespeed.web.dev/analysis/https-pro-dek-co-uk/ppy3rmyotr?form_factor=desktop](https://pagespeed.web.dev/analysis/https-pro-dek-co-uk/ppy3rmyotr?form_factor=desktop)

**Comment by lensbreak** [comment]
Two versions of megamenu have passed and you can still not have more than one mega menu next to each other if its set to open on hover. The first menu works, any other separate mega menu will open the content but you cant hover to the content, it immediately disappears.

**Comment by nicholaszein** [comment]
Hello guys!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
